jme3-android/src/main/java/com/jme3/audio/plugins/NativeVorbisFile.java

```diff
@@ -16,22 +16,33 @@ public class NativeVorbisFile {
     
     static {
         System.loadLibrary("decodejme");
-        nativeInit();
+        preInit();
     }
     
-    public NativeVorbisFile(int fd, long off, long len) throws IOException {
-        open(fd, off, len);
+    /**
+     * Initializes an ogg vorbis native file from a file descriptor [fd].
+     * 
+     * @param fd an integer representing the file descriptor 
+     * @param offset an integer representing the start of the 
+     * @param length an integer representing the length of the 
+     * @throws IOException
+     */
+    public NativeVorbisFile(int fd, long offset, long length) throws IOException {
+        init(fd, offset, length);
     }
     
-    private native void open(int fd, long off, long len) throws IOException;
+    private native void init(int fd, long offset, long length) throws IOException;
```

Comment:
what about naming it "load"? just a thought

Reply:
Hi thanks for the review, yeah i thought of this too, multiple things can work here, for example `getVorbisFromFile`, `createVorbisFromFile`, `loadVorbisFromFile`, but we have to consider the function/method original source code: 
https://github.com/jMonkeyEngine/jmonkeyengine/blob/493195eae810c4a38744e6c34ea0e7e8c48542a9/jme3-android-native/src/native/jme_decode/com_jme3_audio_plugins_NativeVorbisFile.c#L179-L226

What it does here, is it neither opens nor loads something ("load" can mean different things, including something precreated, cached and now scheduled to be loaded by a loader), the method here basically creates and initializes a new `OggVorbis_File` from an asset file (it doesn't create only and it doesn't load from cache), i consider it as a low constructor function, so the best name is `init()` as it is also called inside a constructor.

Reply:
In java architecture, constructors are the `<init>()` methods, so this was the idea as the original author calls it inside the constructor.

Reply:
The `nativeInit` also, it doesn't initialize C or native buffers, it retrieves some java fields into the native environment to be used as data manipulators later between java code and C code, so it prepares the java fields into the native environment.

Reply:
I see, I am fine with this, thanks